services/common/src/interfaces/index.ts

```diff
@@ -27,3 +28,9 @@ export * from "./user";
 export * from "./projects";
 export * from "./documentZipTask.interface";
 export * from "./documentZipProgress.interface";
+export * from "./common";
+export * from "./noticeOfWorkApplication.interface";
+export * from "./now";
+export * from "./noticeOfWork.interface";
+export * from "./noticeOfWorkDraftPermit.interface";
+export * from "./search/searchResult.interface";
```

Comment:
Probably not something that needs to be dealt with here, but I just read an article discussing the performance implications of using many nested barrell files like this and how it ends up importing everything any time any of them are used.  Probably not an issue with interfaces and the like though.  With our previous setup being built we kind of needed to rely on this, but with new components and such are we able to use more direct imports?

Reply:
Yeah, going forwards we should use direct imports, as you're correct that it's no longer needed. Added these as it solved a good chunk of errors that I didn't have to deal with 😅